tests/sentry/identity/test_oauth2.py

```diff
@@ -30,17 +38,49 @@ def view(self):
             client_secret="secret-value",
         )
 
-    @responses.activate
```

Comment:
Was responses not able to do the mocking you needed? It should be able to mock out anything using urllib3.

Reply:
@markstory I added asserts to check the arguments passed to `safe_urlopen()` for the token exchange.

Reply:
You can do that with responses as well. `responses.calls` gives you access to all the requests that were made and the url/parameters that were used for each mocked request.

Reply:
@markstory I've updated to `responses.calls`.